### DISCUSSION TITLE
Firefly updated IOTA 2.0.6 App will not open.

### DISCUSSION THREAD

**Discussion by vmillervm** [iotaledger/firefly#8027] [Help & Support]
Installed the new updated IOTA Firefly App, which is linked to my Ledger Plus device, The new update 2.0.6 version will not open. and will not generate my account from the old app.  so, I can't login.

**Comment by begonaalvarezd** [comment]
You should update your ledger app to the latest version 0.8.6 to be compatible but the latest protocol version
